Week2/index.js

```diff
@@ -0,0 +1,92 @@
+const mysql = require('mysql');
+const { promisify } = require('util');
+
+const createHelpOptions = () => {
+  console.log(`Usage: node index.js [options]
+
+Options:
+
+1 .....(country)                  shows answer of "What is the capital of a country?"
+2 .....(region)                   shows answer of "List al the language spoken in a region"
+3 .....(language)                 shows answer of "Find the number of cities in which language
+                                  is spoken."
+4 .....(region) .....(language)   shows answer of "Are there any countries in any region 
+                                  with a specific language as the official language?"
+5                                 shows answer of "List all the continents with the number of
+                                  languages spoken in each continent."
+`);
+};
+createHelpOptions();
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world',
+});
+
+const execQuery = promisify(connection.query.bind(connection));
+
+const showQuery = async (questionNo, answer1, answer2) => {
+  const questionNumber = parseInt(questionNo, 10);
+  if (Number.isNaN(questionNumber)) {
+    console.error('Please enter a number.');
+    return;
+  } else if (questionNumber < 1 || questionNumber > 5) {
+    console.error('Please enter a number between 1 and 5');
+    return;
+  }
+  const queries = [
+    {
+      query: `select city.name as CAPITAL from city join country on 
+      city.id=country.capital where country.name='${answer1}';`,
+      number: 1,
```

Comment:
The query is good but you have to use Prepared statement please (no country.name='${answer1}' but instead country.name= ? to avoid sql injection

Reply:
@mogaleaf  Thank you for your helpful feedback. I made it.